### PR TITLE
Idincer944/issue13

### PR DESCRIPTION
**Pull request by idincer944**
app.js configuratiopn handled. Auth.js file has been written. User model's  created_user and joined_user have been changed accordingly. #13

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1,6 +1,4 @@
 NODE_LOCAL_PORT=3000
 
-MONGODB_USER=admin
-MONGODB_PASSWORD=13SomeSecretPasswordNo1Nos13
-MONGODB_DATABASE=myblog
-DB_URL=mongodb://127.0.0.1:27017/blogs-db
\ No newline at end of file
+DB_URL=mongodb://127.0.0.1:27017/cigkoftes-db
```
Comment (saidbaradai) [inline .env:3]: localhost would be better than 127.0.0.1:27017 :)

**Review by saidbaradai — CHANGES_REQUESTED** [review]
package-lock.js need to be updated

**Review by saidbaradai — APPROVED** [review]
LGTM

**Review by ilaydanurguzel1 — APPROVED** [review]
lgtm

**Pull request closed without merging**